src/videoTrackingHTML5.js

```diff
@@ -0,0 +1,248 @@
+/**
+ * Video tracking for HTML5 videos
+ * 
+ * This script analyses the interactions with <video> elements.
+ */
+//4Patryk: I'm not sure whether to put "subscribe" here at all?
+ export default (subscribe, {eventCategoryLabel,
```

Comment:
when I removed it, the code wouldn't work - clearly it expects this var to be there. I actually could use subscribe, but I didn't want to, as I have additional logic (lines 48-57) running when sending the event, so i wanted to keep it clean, all in one piece.

Reply:
It fails because when you call that function (https://github.com/PiwikPRO/solutions-templates/pull/21/files#diff-2a2150d0b57522a5f463ffea155f57cc31bf3439fc4953b00b0bca842e89970bR254) you provides that argument, however it is still dumb function which can be removed so `subscribe` can be removed also.

Reply:
I see, in such a case I removed it in both places now and test that it works properly, branch updated.